st2client/st2client/formatters/table.py

```diff
@@ -248,3 +248,13 @@ def _get_attribute_value(subject, attribute):
         if isinstance(r_val, list) or isinstance(r_val, dict):
             return r_val if len(r_val) > 0 else ''
         return r_val
+
+
+class SingleRowTable(object):
+
+    @staticmethod
+    def note_box(message):
+        note = PrettyTable([""])
+        note.header = False
+        note.add_row([message])
+        print note
```

Comment:
Minor API consistency thing - all other formatter class methods return a value instead of printing it so it would be better for this one to do that as well.